cylc/flow/scripts/workflow_state.py

```diff
@@ -35,14 +38,12 @@
 Legacy (pre-8.3.0) options are supported, but deprecated, for existing scripts:
   cylc workflow-state --task=NAME --point=CYCLE --status=STATUS
       --output=MESSAGE --message=MESSAGE --task-point WORKFLOW
-(Note from 8.0 until 8.3.0 --output and --message both match task messages).
+(Note from 8.0 to 8.3.0 --output and --message both matched task messages).
 
 In "cycle/task:selector" the selector will match task statuses, unless:
-  - if it is not a known status, it will match task output triggers
-    (Cylc 8 DB) or task ouput messages (Cylc 7 DB)
-  - with --triggers, it will only match task output triggers
-  - with --messages (deprecated), it will only match task output messages.
-    Triggers are more robust - they match manually and naturally set outputs.
+  - with --triggers, it will only match task output triggers.
+  - with --messages, it will only match task output messages. DEPRECATED
+    use triggers instead - they match manually and naturally set outputs.
```

Comment:
Not deprecated though? 
```suggestion
  - with --messages, it will only match task output messages. It is recommended
    to use triggers instead - they match manually and naturally set outputs.
```

Reply:
It is deprecated, isn't it? We support querying output messages for back-compatibility, but there's no good reason to choose messages over triggers now, so we should eventually drop the option.

Reply:
There's no deprecation warning with the `--messages` option?

Reply:
Hmm. Maybe it should have one.  Not sure if I imagined the deprecation, just thought it should be deprecated, or got it from Oliver's early tidy-up PR - ping @oliver-sanders ?

Reply:
I think the deprecated applies to querying by task state, not message.

Querying by task state can be flaky, it would be reasonable to deprecate this.

Querying by task message isn't much different to querying by task trigger. It might be useful if the message isn't registered as an output.

Reply:
Fair enough, taking the not-deprecated tweak.